.circleci/config.yml

```diff
@@ -66,14 +66,37 @@ jobs:
               sleep 5
           done
           if [[ -f "$FILE_DIST" ]]; then
+            echo "Precompiled version of web app found"
             mkdir dist && tar -xf $FILE_DIST -C dist --strip-components=1
           else
-            npm ci --silent && make build
+            echo "Building web app from source"
+            make dist
           fi
       - persist_to_workspace:
           root: /mnt/ramdisk
           paths:
             - mattermost-webapp
+  setup-focalboard-product:
+    working_directory: /mnt/ramdisk/
+    docker:
+      - image: cimg/go:1.18-node
+    resource_class: xlarge
```

Comment:
Do we need  8 CPU Cores, and 16GB of RAM for this? 
If not let's lower this to something like `large` or `medium` 

https://circleci.com/docs/configuration-reference/#docker-execution-environment

Reply:
I can lower it. I'm not sure what the requirements should be since we're not very consistent with the machine size we use throughout this file (like `check-mocks` uses an xlarge, `check-email-templates` uses a medium+, and `check-store-layers` uses a medium even though they seem like they'd be the same).